pkg/signing/signing.go

```diff
@@ -89,15 +87,64 @@ func sifAddSignature(fimg *sif.FileImage, groupid, link uint32, fingerprint [20]
 	return nil
 }
 
+// Copy-paste from sylabs/sif
+// datatypeStr returns a string representation of a datatype.
+func datatypeStr(dtype sif.Datatype) string {
```

Comment:
this is usually done by defining a `String()` method on the type, `sif.Datatype` in this case.

The copy/paste comment makes me guess this is already there?

Reply:
This function already exists here https://github.com/sylabs/sif/blob/master/pkg/sif/fmt.go#L61, since that non-exported (for now...) I just copy-paste here.

Since this function is not _that_ importent (it only generates a name for `sylog.Debugf()`), I can leave it out until we have the `String()` method for `Datatype`. I'm fine either way.

Reply:
I think this is OK for now. I think it makes sense to support a `String()` method for data types within the SIF package. Lets open an issue for that in the SIF repo and integrate it the next time we cut a SIF release

Reply:
Issue opened -> https://github.com/sylabs/sif/issues/67